package.json

```diff
@@ -25,7 +25,7 @@
     "extends browserslist-config-terra"
   ],
   "eslintConfig": {
-    "extends": "terra",
+    "extends": "@cerner/terra",
     "overrides": [
```

Comment:
Just released this when looking at something else, but can't these overrides be removed?
https://github.com/cerner/terra-toolkit/blob/master/packages/eslint-config-terra/eslint.config.js#L81-L87

Reply:
Do you mean the eslint overrides in both terra-toolkit and terra-core or just terra-toolkit?